app/(auth)/register.tsx

```diff
@@ -1,54 +1,75 @@
-import { Formiz, useForm, useFormFields } from '@formiz/core';
+import { Formiz, useForm } from '@formiz/core';
 import { FieldInput } from '@/components/FieldInput';
 import { focus } from '@/utils/formUtils';
-import { useRef } from 'react';
+import { useRef, useState } from 'react';
 import { TextInput } from 'react-native';
-import { Button, Stack } from 'react-native-ficus-ui';
-import { isEmail, isMinLength } from '@formiz/validations';
+import {
+  Box,
+  Button,
+  Icon,
+  Modal,
+  Stack,
+  Text,
+  TouchableOpacity,
+} from 'react-native-ficus-ui';
+import { isEmail } from '@formiz/validations';
 import { useCreateAccount } from '@/modules/account/account.service';
 import { useRouter } from 'expo-router';
 import { useToast } from '@/modules/toast/useToast';
+import { useDarkMode } from '@/theme/useDarkMode';
+import { CardStatus } from '@/components/CardStatus';
+
+const CardWarningRegister = () => {
+  const router = useRouter();
+  const { colorModeValue } = useDarkMode();
+  return (
+    <CardStatus type="warning" title="Demo mode" mt="md">
+      <Box flexDirection="row" flexWrap="wrap">
+        <Text fontSize="lg" color={colorModeValue('gray.800', 'gray.50')}>
+          This is a read-only demo, but you can Sign in to test some of the
+          features. Just remember, no changes can be made. Enjoy the features!
+        </Text>
+        <TouchableOpacity onPress={() => router.push('/login')}>
+          <Text
+            fontSize="lg"
+            fontWeight="700"
+            color={colorModeValue('gray.800', 'gray.50')}
+            textDecorationLine="underline"
+          >
+            Sign in
+          </Text>
+        </TouchableOpacity>
+      </Box>
+    </CardStatus>
+  );
+};
 
 const Register = () => {
   const router = useRouter();
   const { showError, showSuccess } = useToast();
-
-  const passwordRef = useRef<TextInput>(null);
-  const confirmPasswordRef = useRef<TextInput>(null);
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const nameRef = useRef<TextInput>(null);
 
   const { mutate: createAccount, isLoading } = useCreateAccount({
     onSuccess: () => {
       router.replace('/login');
       showSuccess('You account has been created with success, you can login');
     },
-    onError: (error) => {
-      if (error?.response?.data?.errorKey === 'emailexists') {
-        showError('This mail address is already used by another user');
-      } else {
-        showError(
-          'An error occured during your registration, please try again'
-        );
-      }
+    onError: () => {
```

Comment:
I do not agree here, here you show the "Demo" error like the error will always be because of a demo mode

I think the app should not be aware of the mode of start-ui-web, details